mapactionpy_arcmap/arcmap_runner.py

```diff
@@ -126,28 +140,79 @@ def getOrientation(countryName):
         print("Error: Could not derive country extent from " + url)
         print("Exiting.")
         sys.exit(1)
-           
+
 
 def main(args):
     args = parser.parse_args()
-    cookbookFile = args.cookbookFile
-    layerPropertiesFile = args.layerConfig
+
+    # Construct a Crash Move Folder object if the cmf_description.json exists
     crashMoveFolder = args.crashMoveFolder
-    layerDirectory = args.layerDirectory
+    cmfFilePath = os.path.join(crashMoveFolder, "cmf_description.json")
+    eventFilePath = os.path.join(crashMoveFolder, "event_description.json")
+
+    cmf = None
+    event = None
+
+    if os.path.exists(cmfFilePath):
+        cmf = CrashMoveFolder(cmfFilePath)
+        event = Event(eventFilePath)
+
     productName = args.productName
-    countryName = args.countryName
 
+    # If no country name supplied, need to find it from the event_description.json
+    if args.countryName:
```

Comment:
I think that checking that the Country Name and ISO3 code correspond is something that should go in the constructor for the Event class.